src/lib/y2system_role_handlers/worker_role_finish.rb

```diff
@@ -57,7 +63,14 @@ def configure_salt_minion(master)
       master_conf.save
     end
 
-    def configure_systemd_timesync(master)
+    #
+    # Save the timesyncd configuration
+    #
+    # @param servers [Array<String>,nil] the NTP servers
+    def configure_systemd_timesync(servers)
+      return if servers.nil? || servers.empty?
+      log.info("Configured NTP servers: #{servers.inspect}")
```

Comment:
can you please switch lines? So it logs even empty list and nil? IT is quite helpful for debugging.

Reply:
Done